### DISCUSSION TITLE
Dashboards?

### DISCUSSION THREAD

**Discussion by phortx** [naymspace/backpex#372] [General]
Hi, I'm using currently kaffy and consider to switch over to backpex. Is there a dashboard feature in planning?

**Comment by pehbehbeh** [comment]
We do not have dedicated dashboard features (yet). However, it is very easy to extend Backpex with custom LiveViews. 

You basically add another route to your own LiveView, reuse the same base layout and add some stuff to it (maybe using daisyUI components). Done. 

What features do you have in mind?

**Comment by phortx** [comment]
I think the main advantage of kaffy here is, that you can simply define a function in the configuration which outputs a dashboard panel configuration and you have a dashboard with metrics without much setup work. Would be awesome so have a similar feature in backpex.

See https://github.com/aesmail/kaffy?tab=readme-ov-file#dashboard-page
